specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-02-01/imds.json

```diff
@@ -0,0 +1,769 @@
+{   
+    "swagger": "2.0",
+    "info": {
+        "title": "InstanceMetadataClient",
+        "description": "The Azure Instance Metadata Client",
+        "version": "2019-02-01"
+    },
+    "host": "169.254.169.254",
+    "basePath": "/metadata",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/instance": {
+            "get": {
+                "operationId": "Instances_GetMetadata",
+                "description": "Get Instance Metadata for the Virtual Machine.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/MetadataParameter"
+                    }   
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Instance"
+                        }
+                    },
+                    "400": {
```

Comment:
Any status codes added here will cause codegen to treat them as successes, is that intentional?  If not please remove all non-success status codes (comment applies to all operations).

Reply:
How are these potential responses supposed to be treated? The previous schema version 2018-10-01 had a similar list of potential responses. This new schema version is extremely similar.

Reply:
Any response code not listed here will be handled by the `default` clause which will treat it as an error.  And if the error response body is ODataV4 compliant the AutoRest codegen will propagate that in the error response (exception in C# etc).